Sources/MobileSdk/MDoc.swift

```diff
@@ -168,4 +169,6 @@ public enum BLESessionState {
     case uploadProgress(Int, Int)
     /// App should display a success message and offer to close the page
     case success
+    /// App should display a canceled message
+    case canceled
```

Comment:
What's the use case for this? In my mind, if the app has a cancel button it will know to kill the whole thing and won't need a callback to know that the cancel function was called?

Reply:
This is to make it easier to create a component and avoid passing cancel properties to children, but not necessary.